### PR TITLE
Add option on conf file

### PR DESCRIPTION
**Pull request by OliPou**
Hi, 

I add couple of parameters needed to custom a a configuration backup backup.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ---
 keepalived_install: False
 keepalived_packages: "{{ __keepalived_packages }}"
+vrrp_script.weight_enabled: true
+vrrp_script.fall_enabled: true
+vrrp_script.rise_enabled : true
```
Comment (tleguern) [inline defaults/main.yml]: I am firmly against adding such variables, it tends to clutter roles uselessly. A better approach is to remove the default value.

File: README.md [hunk 2]
```diff
@@ -92,6 +97,32 @@ Example Playbook
...
+           interface: eth0
+           track_interface:
+             - eth0
+           virtual_ipaddresses:
+             - "{{ vip_front + " dev" " eth0" }}"
+           track_script: haproxy
+      roles:
+      - role: ansible-keepalived
+```
+
```
Comment (tleguern) [inline README.md]: Looks like this part was added by mistake.

**Review by tleguern — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (OliPou) [reply defaults/main.yml]: I just removed them and removed the default value. Is it all right for you now?

Comment (tleguern) [reply defaults/main.yml]: Yes, much better.

**Review by tleguern — APPROVED** [review]

**Comment by tleguern** [comment]
I am going to simplify the role's configuration in order to remove these pesky variables, but thanks for your contribution.

**Pull request closed without merging**